skins/cat17/src/sass/layouts/pages/privacy_protection.scss

```diff
@@ -0,0 +1,7 @@
+.page-privacy_protection {
+	.content {
+		li {
+			list-style-position: inside;
```

Comment:
The list at the top of the page looks ok, but when you have longer lines, they are not indented correctly when you have a line break in the list (see section "Betroffene Personengruppen und Daten oder Datenkategorien")

Reply:
Looks pretty much like in 10h16 on prod - please define "correctly" (T179936).

Reply:
padding-left: 1.5em;
    text-indent: -1.5em;

looks nice